laos_gggi/co2_processing.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+import os
+
+
+def process_co2():
+    if not os.path.isfile("../data/co2.csv"):
+        url = "https://gml.noaa.gov/aftp/products/trends/co2/co2_mm_gl.csv"
+        df = pd.read_csv(url, skiprows=38)
+        df["day"] = 1
+        df["Date"] = pd.to_datetime(dict(year=df.year, month=df.month, day=df.day))
+        df = df.set_index("Date")
```

Comment:
Use the `inplace=True` argument rather than re-assigning. It doesn't matter here, but for huge datasets it's more memory efficient.